ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -296,13 +299,13 @@ struct ZipBuilderTool: ParsableCommand {
                                                isVersionCheckEnabled: carthageVersionCheck)
       }
 
-      FirebaseBuilder(zipBuilder: builder).build(in: projectDir,
-                                                 minimumIOSVersion: minimumIOSVersion,
+      FirebaseBuilder(zipBuilder: builder).build(templateDir: paths.templateDir,
                                                  carthageBuildOptions: carthageOptions)
     }
 
     if !keepBuildArtifacts {
-      FileManager.default.removeIfExists(at: projectDir.deletingLastPathComponent())
+      let tempDir = FileManager.default.temporaryDirectory(withName: "random")
```

Comment:
Should this be "random" or something else?

Reply:
Renamed to "placeholder". It seems like there should b a better way to delete the tempDir without generating the name, but I couldn't find it.